index.html

```diff
@@ -30,6 +30,126 @@ <h1>Hey there. I'm John Buhendwa <span>I'm a software developer</span></h1>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-section">
+            <h2>About me</h2>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <button>Get My Resume</button>
```

Comment:
- [ ] **The Get My Resume** button here is also lacking the button styling respective to hover and pressed interactions by a user please do well to fix this as well. You must stick to the design as much as possible :wink: